PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterNonLinearityMCAfterburner.h

```diff
@@ -0,0 +1,62 @@
+#ifndef ALIEMCALCORRECTIONCLUSTERNONLINEARITYMCAFTERBURNER_H
+#define ALIEMCALCORRECTIONCLUSTERNONLINEARITYMCAFTERBURNER_H
+
+#include "AliEmcalCorrectionComponent.h"
+
+/**
+ * @class AliEmcalCorrectionClusterNonLinearityMCAfterburner
+ * @ingroup EMCALCOREFW
+ * @brief Cluster energy non-linearity correction MC afterburner in the EMCal correction framework.
+ *
+ * Non-linearity correction to the MC cluster energy is necessary because the current non linearity is done for
+ * 50MeV cell threshold and all analyses use a 100MeV cell threshold for the clusterization.
+ * Thus, the MC need to be corrected in a second step
+ 
+ The energy of the cluster **after** the MC non-linearity correction can be retrieved using the method `cluster->GetNonLinCorrEnergy()`.
+ *
+ * Based on code in AliCaloPhotonCuts.cxx.
+ *
+ * @author 15 active people originally in AliCaloPhotonCuts.cxx
+ * @author Eliane Epple, Yale University, include the PCM aproach into the EMCal correction FW using correction components
+ * @date Nov 13, 2018
+ */
```

Comment:
Please also add an entry in the main correction framework docs (`PWG/EMCAL/READMEemcCorrections.txt`) to link to this class and provide a one sentence description (particularly to contrast it from the existing non-linearity)

Reply:
Done